plugins/woocommerce/src/Admin/Notes/DeprecatedNotes.php

```diff
@@ -223,27 +223,6 @@ class WC_Admin_Notes_Giving_Feedback_Notes extends DeprecatedClassFacade {
 	protected static $deprecated_in_version = '4.8.0';
 }
 
-/**
- * WC_Admin_Notes_Insight_First_Sale.
- *
- * @deprecated since 4.8.0, use InsightFirstSale
- */
-class WC_Admin_Notes_Insight_First_Sale extends DeprecatedClassFacade {
```

Comment:
It looks like this was deprecated back in 4.8.0 and probably safe to remove at this point, but just want to check on the original reasons for this and make sure we're not breaking anything.  cc @moon0326